sql/src/lib.rs

```diff
@@ -136,11 +140,16 @@ impl QueryBuilder for SQLiteQueryBuilder {
             &Boolean(v) => self.push_sql(if v { "1" } else { "0" }),
             &Long(v) => self.push_sql(v.to_string().as_str()),
             &Double(OrderedFloat(v)) => self.push_sql(v.to_string().as_str()),
-
-            // These are both `Rc`. We can just clone an `Rc<String>`, but we
-            // must make a new single `String`, wrapped in an `Rc`, for keywords.
-            &String(ref s) => self.push_static_arg(s.clone()),
-            &Keyword(ref s) => self.push_static_arg(Rc::new(s.as_ref().to_string())),
+            // These are both `Rc`. Unfortunately, we can't use that fact when
```

Comment:
Yeah, this is awkward.  It does seem that you could rework this to use `rusqlite::ToSqlOutput<'a>`, since the `Rc` will have at least as long a lifetime as `&'a self`.  But it's work, and I'm not 100% sure I'm actually correct.